doc/user/installation.rst

```diff
@@ -79,9 +79,23 @@ options from the list below.
 
 .. program:: configure
 
+.. option:: --enable-tcmalloc
+
+   Enable the alternate malloc library.  In some cases this is faster and more efficient,
+   in some cases it is not.
+
+.. option:: --disable-doc
+
+   Do not build any documentation, especially this one.
```

Comment:
Why `especially this one`, lol

Reply:
how about: s/especially/including

Reply:
@qlyoung because it was 10 o'clock at night and I was joking.  Changed to including.